src/ZoweJobNode.ts

```diff
@@ -140,11 +119,6 @@ export class Job extends vscode.TreeItem {
         this.dirty = false;
         return this.children;
     }
-    public getDetailLabel(): string {
-        return this.contextValue === "Job" || this.contextValue === extension.JOBS_JOB_CONTEXT + extension.FAV_SUFFIX ?
-            `${this.job.jobname}(${this.job.jobid})`
-            : Job.createSearchLabel(this.owner, this.prefix, this.searchId);
-    }
```

Comment:
I can see `createSearchLabel` being moved under `ZosJobsProvider.ts`, but maybe I missed this function. (Or maybe it isn't used anymore?

Reply:
getDetailLabel isn't referenced. Actually part of a commented out block which I should have removed/cleaned up